### PR TITLE
fix a ImageExportPlugin test

### PR DESCRIPTION
**Pull request by georgehrke**
fixes one test that was commented out so far.

One test remains commented out, blocked by https://github.com/fruux/sabre-vobject/issues/325#issuecomment-227690127

### CODE REVIEW

File: apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php [hunk 1]
```diff
@@ -140,11 +140,8 @@ public function providesPhotoData() {
...
 			'vcard 3 with PHOTO' => [['Content-Type' => 'image/jpeg', 'body' => '12345'], "BEGIN:VCARD\r\nVERSION:3.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO;ENCODING=b;TYPE=JPEG:MTIzNDU=\r\nEND:VCARD\r\n"],
-			//
-			// TODO: these three below are not working - needs debugging
-			//
-			//'vcard 3 with PHOTO URL' => [['Content-Type' => 'image/jpeg', 'body' => '12345'], "BEGIN:VCARD\r\nVERSION:3.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO;TYPE=JPEG:http://example.org/photo.jpg\r\nEND:VCARD\r\n"],
-			//'vcard 4 with PHOTO' => [['Content-Type' => 'image/jpeg', 'body' => '12345'], "BEGIN:VCARD\r\nVERSION:4.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO:data:image/jpeg;MTIzNDU=\r\nEND:VCARD\r\n"],
+			'vcard 3 with PHOTO URL' => [['Content-Type' => 'image/jpeg', 'body' => 'http://example.com/photo.jpg'], "BEGIN:VCARD\r\nVERSION:3.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO;TYPE=JPEG;VALUE=URI:http://example.com/photo.jpg\r\nEND:VCARD\r\n"],
```
Comment (DeepDiver1975) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: returning the url as body seems wrong

Comment (georgehrke) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: But do we really won't to load arbitrary data from the net and just deliver it to the user?

Comment (LukasReschke) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: No.

Comment (DeepDiver1975) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: > But do we really won't to load arbitrary data from the net and just deliver it to the user?

no - let's just return false on getPhoto in these cases

Comment (georgehrke) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: okay, will change the behavior accordingly

File: apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php [hunk 2]
```diff
@@ -140,12 +140,9 @@ public function providesPhotoData() {
...
-			//'vcard 4 with PHOTO' => [['Content-Type' => 'image/jpeg', 'body' => '12345'], "BEGIN:VCARD\r\nVERSION:4.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO:data:image/jpeg;MTIzNDU=\r\nEND:VCARD\r\n"],
-			'vcard 4 with PHOTO URL' => [['Content-Type' => 'image/jpeg', 'body' => 'http://example.org/photo.jpg'], "BEGIN:VCARD\r\nVERSION:4.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO;MEDIATYPE=image/jpeg:http://example.org/photo.jpg\r\nEND:VCARD\r\n"],
+			'vcard 3 with PHOTO URL' => [false, "BEGIN:VCARD\r\nVERSION:3.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO;TYPE=JPEG;VALUE=URI:http://example.com/photo.jpg\r\nEND:VCARD\r\n"],
+			//'vcard 4 with PHOTO' => [['Content-Type' => 'image/jpeg', 'body' => '12345'], "BEGIN:VCARD\r\nVERSION:4.0\r\nPRODID:-//Sabre//Sabre VObject 3.5.0//EN\r\nUID:12345\r\nFN:12345\r\nN:12345;;;;\r\nPHOTO:data:image/jpeg;base64,MTIzNDU=\r\nEND:VCARD\r\n"],
```
Comment (DeepDiver1975) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: what about this one?

Comment (georgehrke) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: see https://github.com/fruux/sabre-vobject/issues/325#issuecomment-227690127

**Comment by PVince81** [comment]
:+1:

**Comment by georgehrke** [comment]
We should fix the `vcard 4 with PHOTO` test before merging

File: apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php (hunk 2 above)
Comment (georgehrke) [inline apps/dav/tests/unit/CardDAV/ImageExportPluginTest.php]: done

**Comment by georgehrke** [comment]
please review @DeepDiver1975 @ChristophWurst

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**